main.nf

```diff
@@ -2106,6 +2106,34 @@ process damageprofiler {
     """
 }
 
+// Calculate typical aDNA damage frequency distribution with mapDamage
+
+process mapdamage_estimation {
```

Comment:
Maybe `mapdamage_profiling` would be clearer? But not important

Reply:
the section is called Damage Calculation. so could change to that. tho i doubt it matters much